src/EfficientDynamoDb/DynamoDbContextMetadata.cs

```diff
@@ -120,7 +120,7 @@ private DdbConverter GetOrAddCachedConverter(Type propertyType, Type? converterT
                 return converter;
             
             var nullableConverterType = typeof(NullableValueTypeDdbConverter<>).MakeGenericType(type);
-            return (DdbConverter) Activator.CreateInstance(nullableConverterType, converter);
+            return (DdbConverter) Activator.CreateInstance(nullableConverterType, converter)!;
```

Comment:
Activator.CreateInstance returns null only for `Nullable<T>` types, e.g. `int?`, so  should be safe to suppress here.

Reply:
I think it's worth having this explanation as a comment in code. Do you mind adding it?

Reply:
Added